### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by tamasgt** [bilal-fazlani/commanddotnet#434] [Q&A] [answered]
Hello,

I'm new here. Particularly good the CommandDotNet, I tried several things, and everything works. I have a problem with localization. How to use this correctly?

In the documentation:
[https://commanddotnet.bilal-fazlani.com/localization/overview/](https://commanddotnet.bilal-fazlani.com/localization/overview/)

```csharp
class MyResources : Resources
{
    public override string Error_File_not_found(string fullPath) => $"missing file: {fullPath}";
    ...
}

static int Main(string[] args)
{
    return new AppRunner<ValidationApp>(settings, new MyResources()).Run(args);
}
```
but **AppRunner<Program>' does not contain a constructor that takes 2 arguments**

And there is no such setting, it does not appear:  `appRunner.UseCultureDirective()`
https://commanddotnet.bilal-fazlani.com/localization/culture-directive/#culture-directive_1

What am I using wrong? Thanks,
Tamas

**Comment by drewburlingame** [comment, marked as answer]
Which version are you using?

**Reply by drewburlingame** [reply]
Ah, I see the problem.  The documentation is inaccurate atm.  It reflected a branch I hadn't pushed yet and cannot at the moment.  To resolve this for now, you'll need to use `return new AppRunner(typeof(ValidationApp), settings, new MyResources()).Run(args);`

Thanks for pointing this out.

**Reply by tamasgt** [reply]
Thanks, it works.
